src/Helper/ProcessHelper.php

```diff
@@ -78,7 +78,7 @@ public function runCommand($command, $allowFailures = false, $callback = null)
         $process->run($callback);
 
         if (!$process->isSuccessful() && !$allowFailures) {
-            throw new \RuntimeException($process->getErrorOutput());
+            throw new \RuntimeException(trim($process->getErrorOutput()), $process->getExitCode());
```

Comment:
There isn't any exception suitable for this AFAIK. Current exceptions extending directly `\RuntimeException` are:
* `AdapterException`
* `FileNotFoundException`
* `InvalidStateException`
* `NotImplementedException`
* `UnknownRemoteException`
* `UnsupportedTypeException`